make.sh

```diff
@@ -38,6 +38,25 @@ function echo_bold() {
     echo -e "\e[1m$1\e[0m"
 }
 
+echo_stage "Checking need to build a new version..."
+BUILD_COMMIT_ID_FILE=build/__commit_id__
+if [[ -f $BUILD_COMMIT_ID_FILE ]]; then
+    # Query tag's commit:
+    REQUESTED_BUILD_VERSION_COMMIT=`git ls-remote $LIGHTGBM_REPO_URL $LIGHTGBM_VERSION | cut -f1`
+    # Queries upon commit ids have no output. Assume it is a commit:
```

Comment:
I think you should type `query`, otherwise, you should comment `queries` in the above commit.